joomla-extensions/weblinks
# Language files

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/16753

### Summary of Changes
Moves the language files from the global language folder to the component/module specific.
During install and update, the en-GB files in the global folder are deleted.

### Testing Instructions
Install the component and make sure the language files in the global language folders are deleted. Test that the strings in Weblinks still are coming in english and not the "untranslated" strings (eg not COM_WEBLINKS_FOO)


### Expected result
Language files served from the extension and not from core


### Actual result
Language files are served by core


### Documentation Changes Required
None
